src/core_plugins/kibana/public/visualize/index.js

```diff
@@ -23,6 +23,14 @@ import { VisualizeConstants } from './visualize_constants';
 import savedObjectRegistry from 'ui/saved_objects/saved_object_registry';
 import savedVisusalizationProvider from 'plugins/kibana/visualize/saved_visualizations/saved_visualization_register';
 
+import { CreateOrDeleteButton } from './listing/create_or_delete_button';
+
+import uiModules from 'ui/modules';
```

Comment:
If we want to have the import location consistent with how it's done in React+JSX, we should import the components where they are needed.